src/cephadm/cephadm

```diff
@@ -6089,6 +6089,8 @@ def _parse_ipv4_route(out: str) -> Dict[str, Dict[str, Set[str]]]:
         if not m:
             continue
         net = m[0][0]
+        if '/' not in net:  # discard /32 single host nets
```

Comment:
Is this actually the place we want to make a change to fix the issue? This output is often just informational and has been like this for a long time. I feel like we would want this type of check in `infer_mon_network` so we fix the building of the mon networks without potentially affecting anything else.

Reply:
As you pointed out, maybe having this code on the `list_networks` may impact legacy code. I'm moving the code to the `infer_mon_network` so it only affects the new public network parsing code.